File: src/app/shared/service/api/transfer.service.ts
import { HttpClient, HttpRequest } from '@angular/common/http';
import { TransferDTORequest } from '../../model/request/transferDTORequest';
import { environment } from 'src/environments/environment';
import { Injectable } from '@angular/core';

@Injectable({
    providedIn:'root'
})
export class TransferService {
    constructor(
        protected _httpClient: HttpClient
    ) { }

    save(dto: TransferDTORequest) {
        const requ = new HttpRequest('POST', `${environment.urlBase}/transfers/transfer`, dto, {
            reportProgress: true,
            responseType: 'json'
        });
        return this._httpClient.request(requ);
    }
}

File: src/main/java/com/sys/botica/crce/pe/sys_botica/service/TransferService.java
package com.sys.botica.crce.pe.sys_botica.service;

import com.sys.botica.crce.pe.sys_botica.dto.HrefEntityDTO;
import com.sys.botica.crce.pe.sys_botica.dto.request.TransferDTORequest;

public interface TransferService {
	public HrefEntityDTO save(TransferDTORequest transferDTORequest,String token);
}
